knowledge-base/slider-max-parameter-zero-error.md

```diff
@@ -25,7 +25,7 @@ res_type: kb
 I'm using the Telerik Slider, and I can't set Max to 0. When I set the Max parameter to zero, the component throws an error.
 
 ## Error Message
->warning ArgumentException: Telerik.Blazor.Components.TelerikSlider`1[System.Int32] requires the Max parameter.
+``ArgumentException: Telerik.Blazor.Components.TelerikSlider`1[System.Int32] requires the Max parameter.``
```

Comment:
Are the double characters by design here and at the end?

Reply:
Yes, this is how you can use a backtick in a code snippet: `` ` ``